.buildkite/pipeline.yml

```diff
@@ -522,7 +526,6 @@ steps:
             - "requirements/**"
             config:
               label: "🔨 GraphQL"
-              <<: *test-agents
```

Comment:
Did you intend to have `*retries` added here too?

Reply:
Oh wow I also did not intend to delete the test-agents here, good catch!